src/main/java/io/seqera/tower/cli/commands/computeenvs/platforms/AwsBatchForgePlatform.java

```diff
@@ -99,7 +107,7 @@ private ForgeConfig buildForge() {
                 .type(provisioningModel)
                 .maxCpus(maxCpus)
                 .ebsAutoScale(!noEbsAutoScale)
-                .fusionEnabled(fusion)
+                .fusionEnabled(false) // Fusion v1 is deprecated, set this field to false independently of fusionV2
```

Comment:
Even better not even set `.fusionEnabled(false)`

Reply:
removed